src/code/z_en_a_keep.c

```diff
@@ -54,10 +54,8 @@ static ColliderCylinderInit sCylinderInit = {
     { 25, 60, 0, { 0, 0, 0 } },
 };
 
-extern CollisionHeader D_06000730;
-
 static CollisionHeader* D_8011546C[] = {
-    &gUnknown1Col, &gUnknown1Col, &gUnknown4Col, &gUnknown5Col, &gUnknown6Col, &D_06000730,
+    &gUnknown1Col, &gUnknown1Col, &gUnknown4Col, &gUnknown5Col, &gUnknown6Col, 0x06000730,
```

Comment:
what's the deal with this?

Reply:
This actor is gameplay keep and doesn't ever check for any other object.  So its safe to assume that it was left in by mistake.  Since there is no real way to figure out what it is I just removed the symbol.